lib/streamlit/type_util.py

```diff
@@ -338,6 +370,13 @@ def is_keras_model(obj: object) -> bool:
     )
 
 
+def is_list_of_scalars(data: Iterable[Any]) -> bool:
+    """Check if the list only contains scalar values."""
+    # Overview on all value that are interpreted as scalar:
+    # https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_scalar.html
+    return infer_dtype(data, skipna=True) not in ["mixed", "unknown-array"]
+
+
```

Comment:
nit: extra space

Reply:
Do you mean the two lines under the function? I think black enforces two lines under every function